images/cluster-setup/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+WORKDIR /
```

Comment:
Could you overwrite the labels of the base image and set specifics?
Something like [this](https://github.com/openshift-pipelines/pipelines-service/blob/main/images/kcp-registrar/Dockerfile#L12-L26)
If you are short of time I can do it afterwards